VideoLocker/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     // with module updates.
     compile project (':android-iconify-fontawesome')
     compile fileTree(dir: 'libs', include: '*.jar')
+    compile 'org.apache.commons:commons-lang3:3.4'
```

Comment:
OK, so I have [removed][1] the `FastDateFormat` dependency in favor of using Gson 2.5's implementation of an ISO 8601 parser. I have also removed all usages of the Apache Commons Lang 2.4 dependency, which was actually a compile-time only transitive dependency of the RoboGuice annotation preprocessor RoboBlender, and caused a `NoClassDefFoundError` upon runtime usage, as [discussed][2] in #750. The utility classes that we were using have been imported in the project itself.

[1]: https://github.com/edx/edx-app-android/pull/748#discussion_r69355348
[2]: https://github.com/edx/edx-app-android/pull/750#issuecomment-227444716